iframe.html

```diff
@@ -54,7 +54,20 @@
       } = JSON.parse(UrlQueryData);
 
       function sendMessageToRN(msg) {
-        window.ReactNativeWebView.postMessage(JSON.stringify(msg));
+
+        if (window.ReactNativeWebView) {
+          // React Native WebView
+          window.ReactNativeWebView.postMessage(JSON.stringify(msg));
```

Comment:
Note: An interesting quirk is that adding targetOrigin here will break Android native.